vagrant/provisioning/roles/foia/templates/application.properties

```diff
@@ -13,15 +13,15 @@ portal.host={{ external_host }}
 portal.port=
 portal.uiApplicationUrl=https://{{ external_host }}/foia
 portal.uiApplicationHost=https://{{ external_host }}
-portal.id={{ 9999999999999999999999 | random | to_uuid }}
+portal.id={{ foia_portal_id | default(9999999999999999999999 | random | to_uuid) }}
```

Comment:
after accepting this change (assuming we go ahead) the facts file for existing portal sites should be updated to include the `foia_portal_id`, `portal_authenticated_mode` and `foia_portal_authentication_type` facts.